catalog/static/config.js

```diff
@@ -19,4 +21,21 @@ if (window.location.hostname === 'quiltdata.com') {
     // userApi: 'https://api.github.com/user',
     // signOutUrl: '/'
   };
+} else {
+  // TODO: This should come from Docker variables on team instances
```

Comment:
It already does; this file gets completely overwritten by `config.js.tmpl` in Docker images - so just add the team variables there.